README.md

```diff
@@ -157,8 +157,8 @@ SIG-specific GitHub discussions.
 | Android: SDK + Automatic Instrumentation&nbsp;<a id="sig-android" href="#sig-android"><sup>🔗</sup></a> | Tuesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY) | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q) | [calendar-android](https://groups.google.com/a/opentelemetry.io/g/calendar-android) | [Ted Young](https://github.com/tedsuo) |
 | Arrow&nbsp;<a id="sig-arrow" href="#sig-arrow"><sup>🔗</sup></a> | Alternating between Tuesday at 16:00 PT and Thursday at 8:00 PT | [Google Doc](https://docs.google.com/document/d/1z8_Ra-ALDaYNa88mMj1gOZtOpLZLRk0-dZEmDjPmcUs) | [#otel-arrow](https://cloud-native.slack.com/archives/C07S4Q67LTF) | [calendar-arrow](https://groups.google.com/a/opentelemetry.io/g/calendar-arrow) | [Trask Stalnaker](https://github.com/trask) |
 | Browser&nbsp;<a id="sig-browser" href="#sig-browser"><sup>🔗</sup></a> | Thursday at 8:30 PT | [Google Doc](https://docs.google.com/document/d/1G3yBAV5SzWBO3mmsBUos5bsn4yxsqkS1MLWl1urVojQ) | [#otel-browser](https://cloud-native.slack.com/archives/C093P0AMP0T) | [calendar-browser](https://groups.google.com/a/opentelemetry.io/g/calendar-browser) | [Ted Young](https://github.com/tedsuo) |
-| Collector&nbsp;<a id="sig-collector" href="#sig-collector"><sup>🔗</sup></a> | Alternating between Tuesday at 17:00 PT, Wednesday at 09:00 PT, and Wednesday at 05:00 PT | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Pablo Baeyens](https://github.com/mx-psi) |
-| C++: SDK&nbsp;<a id="sig-cpp" href="#sig-cpp"><sup>🔗</sup></a> | Alternating between Monday at 13:00 PT and Wednesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw) | [#otel-cpp](https://cloud-native.slack.com/archives/C01N3AT62SJ) | [calendar-cpp](https://groups.google.com/a/opentelemetry.io/g/calendar-cpp) | [Severin Neumann](https://github.com/svrnm) |
+| Collector&nbsp;<a id="sig-collector" href="#sig-collector"><sup>🔗</sup></a> | Alternating between Tuesday at 17:00 PT, Wednesday at 09:00 PT, and Wednesday at 05:00 PT | [Google Doc](https://docs.google.com/document/d/1AeWpwxjMPm8jOAZymCB3BrwwmNiJiKOO7dc4EvVq3x0) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Pablo Baeyens](https://github.com/mx-psi) |
```

Comment:
Should the [archive doc](docs.google.com/document/d/1un2i9RmyCJ6J5cs7MoBjwi1GXAN2DLMgu3Z-0kViN70) also be moved, is it in the correct place? The archive doc has historical meeting notes from July 2019 to March 2024.